developer-resources/Developer-Guidelines.md

```diff
@@ -162,6 +162,15 @@ code is committed into git.
 
 * For java unit tests use /src/test/java folder of the module (standard maven location), and postfix java file names with "Test".
 
+### Working with Dates
+
+Egeria manages date/time instances as absolute Unix epoch timestamp and always stores them as `java.lang.Long`.
```

Comment:
I am not sure about this sentence - when you say store - do you mean the omrs repository connectors that we supply, i.e. in memory and JanusGraph and Crux. 
I see the following
```
public class InstanceProperties extends InstanceElementHeader
{
    private static final long    serialVersionUID = 1L;

    private Date                                effectiveFromTime = null;
    private Date                                effectiveToTime = null;

```

So for these properties the connector is supplied with a Date not a long. The InMemoryOMRSMetadataStore stored the EntityDetail as a java object.

How about "The Egeria OMRS layer handles date/time as either java.lang.long's or as java.util.Date objects. It does not store localised versions of the date / time.

Reply:
maybe also specifically state that they are milliseconds since the epoch (Jan 1 1970)  as in some other areas only seconds are used.